packages/pwa-buildpack/lib/cli/generate-build-report.js

```diff
@@ -0,0 +1,240 @@
+const { existsSync, readFileSync } = require('fs');
+const { parse } = require('@yarnpkg/lockfile');
+const os = require('os');
+const path = require('path');
+const { spawnSync } = require('child_process');
+const https = require('https');
+const fetch = require('node-fetch');
+const agent = new https.Agent({
+    rejectUnauthorized: false
+});
+
+const fetchWithAgent = async url => {
+    return await fetch(url, { agent });
+};
+
+const prettyLogger = require('../util/pretty-logger');
+const { loadEnvironment } = require('../Utilities');
+
+const { uniqBy } = require('lodash');
+const { sampleBackends: defaultSampleBackends } = require('./create-project');
+
+const removeDuplicateBackends = backendEnvironments =>
+    uniqBy(backendEnvironments, 'url');
+
+const fetchSampleBackendUrls = async () => {
+    try {
+        const res = await fetch(
+            'https://fvp0esmt8f.execute-api.us-east-1.amazonaws.com/default/getSampleBackends'
+        );
+        const { sampleBackends } = await res.json();
+
+        return removeDuplicateBackends([
+            ...sampleBackends.environments,
+            ...defaultSampleBackends.environments
+        ]).map(({ url }) => url);
+    } catch {
+        return defaultSampleBackends.environments.map(({ url }) => url);
+    }
+};
+
+const cwd = process.cwd();
+
+function inspectDependencies(package) {
+    prettyLogger.info('Inspecting Dependencies');
+
+    // Inspect all '@magento/...' dependencies.
+    const magentoDependencies = [
```

Comment:
@sirugh while writing tests for this file, it hit me, we should be capturing all `adobe` and `magento` packages right, not just `magento`. What do you think?

Reply:
Depends on what you really want to glean from this information. I think we have at least one adobe dependency so might as well throw that in :)

Reply:
We actually use two:

@adobe/adobe-client-data-layer
@adobe/apollo-link-mutation-queue